workers/unity/Packages/com.improbable.gdk.core/Logging/ForwardingDispatcher.cs

```diff
@@ -11,12 +11,31 @@ namespace Improbable.Gdk.Core
     /// </summary>
     public class ForwardingDispatcher : ILogDispatcher
     {
-        public Connection Connection { get; set; }
-
         private readonly LogLevel minimumLogLevel;
 
         private bool inHandleLog;
 
+        private bool handlingUnityLogMessages;
+        private Connection connection;
+
+        public Connection Connection
+        {
+            get => connection;
+            set
+            {
+                if (connection == null)
+                {
+                    handlingUnityLogMessages = true;
+                    Application.logMessageReceived += LogCallback;
```

Comment:
doesn't this mean that we won't log it to the Unity console anymore, if we don't have a connection?